src/components/Invoice.tsx

```diff
@@ -8,108 +8,171 @@ import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import IconButton from '@mui/material/IconButton';
+import { SubmitHandler, useForm, Controller } from 'react-hook-form';
+
+
+interface IPersonalData {
+
+  companyName: string | null;
```

Comment:
I would rather keep these as `string` only for simplicity, and set the default values as `''`. Should be easier to work with the values that are never nulls.